driver/configuration/provider_viper_authn.go

```diff
@@ -53,6 +58,18 @@ func (v *ViperProvider) AuthenticatorNoopIsEnabled() bool {
 
 }
 
+func (v *ViperProvider) AuthenticatorCookieSessionIsEnabled() bool {
+	return viperx.GetBool(v.l, ViperKeyAuthenticatorCookieSessionIsEnabled, false)
+}
+
+func (v *ViperProvider) AuthenticatorCookieSessionSessionStoreOrigin() string {
+	return viperx.GetString(v.l, ViperKeyAuthenticatorCookieSessionSessionStoreOrigin, "", "AUTHENTICATOR_COOKIE_SEESSION_SESSION_STORE_ORIGIN")
```

Comment:
You can omit `AUTHENTICATOR_COOKIE_SEESSION_SESSION_STORE_ORIGIN`. The last parameters are used for backwards compatible changes (e.g. when the config key changes you can add it to that list and it will look for the values and throw potentially a warning).